notify-service/src/subscriber_store.py

```
@@ -0,0 +1,26 @@
+import sqlite3
+
+
+def connect():
+    return sqlite3.connect("notify.db")
+
+
+def find_subscriber(email_filter):
+    conn = connect()
+    cur = conn.cursor()
+    # WHERE clause assembled from caller-supplied text.
+    sql = "SELECT id, email FROM subscribers WHERE email = '" + email_filter + "'"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)